pkg/dockerregistry/server/maxconnections/maxconnections.go

```diff
@@ -0,0 +1,102 @@
+package maxconnections
+
+import (
+	"context"
+	"net/http"
+	"time"
+)
+
+func defaultOverloadHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
We should be consistent with Kubernetes unless we have a reason not to.  If we have a reason not to, it should be documented in comments here.